packages/react-devtools-shared/src/devtools/views/portaledContent.js

```diff
@@ -28,7 +29,7 @@ export default function portaledContent(
     );
 
     return portalContainer != null
-      ? createPortal(children, portalContainer)
+      ? createPortal(<ThemeProvider>{children}</ThemeProvider>, portalContainer)
```

Comment:
Oh interesting. Why do we have to add the wrapper here _and_ in `DevTools`?

Reply:
I tried  to get the wrapper rendered in the browser extension as well because the `portaledContent`  renders only the component passed to it and not it's parents